### ISSUE TITLE
Script fails after performing the action

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#162]
**Expected behavior**
Script should not fail if the action is performed

**Actual behavior**
Even though the submit button is clicked, the script fails. The error given is `Button[type="submit"] not found`. But this is the button that was clicked and navigation to the next page succeeded.

**Steps to replicate**
* Run script in observe mode
`taiko script.js --observe`
```
(async () => {
    try {
        await openBrowser({headless:false});
        await goto("thetrainline.com",{timeout:20000});
 
        await write("London",into(inputField(toRightOf("From"))));
        await press("Enter");
        await write("Manchester");
        await press("Enter");
        await click("Tomorrow")
        await click(button({"type":"submit"}));
        await click("Continue");
    } catch (e) {
        console.error(e);
    } finally {
        closeBrowser();
    }
 })();
```
**Version**
```
commit - 479bd2a0ba1fe6ea82be136b25e1a91a47ad4598
```
Blocks #73

**Comment by sguptatw** [comment]
Another example
```
const { browser, openBrowser, goto, link, click, listItem } = require('taiko');

(async () => {
    try {
        await openBrowser();
        await goto("http://todomvc.com/examples/react/#/");

        await write("to remove");

        await press("Enter");    
        await waitFor(1000);
        await click(checkBox({'class':'toggle'},near('to remove')))
        await hover(checkBox({'class':'toggle'},near('to remove')))

        await click(button({'class':'destroy'},near('to remove')))
        await waitFor(5000);
        await reload("http://todomvc.com/examples/react/#/");

        await closeBrowser();
    } catch (e) {
        console.error(e);
    } finally {
    }
})();
```

**Comment by NivedhaSenthil** [comment]
Fix should be available in 59ca1bd
